piicatcher/command_line.py

```diff
@@ -123,6 +138,12 @@ def cli(
         version: Optional[bool] = typer.Option(
             None, "--version", callback=version_callback, is_eager=True
         ),
+        stats_status: Optional[bool] = typer.Option(
+            None, "--disable-stats", callback=stats_callback, is_eager=True
+        ),
+        enable_stats_status: Optional[bool] = typer.Option(
```

Comment:
Only one of disable or enable is required. Check `log_data` or `log_scan` above. It automatically creates `--no-log-data` or similar. For example, check `--formal` in the [docs](https://typer.tiangolo.com/tutorial/options/help/)

Reply:
This needs separate actions reason being once user disable collection we dont want to collect stats(in respective of he is using `--disable-stats` or not. If we keep 1 option it will get called as true when someone executes without that option(which is possible)